nepi/mixins.py

```diff
@@ -1,18 +1,14 @@
 import json
 
 from django.contrib.auth.decorators import login_required, user_passes_test
-from django.http.response import HttpResponseNotAllowed, HttpResponse, \
-    HttpResponseForbidden
+from django.http.response import HttpResponse, HttpResponseForbidden
 from django.utils.decorators import method_decorator
 
 from nepi.main.models import LearningModule
 
 
```

Comment:
We can test this ajax by doing what Nik did here: https://github.com/ccnmtl/writlarge/pull/2023/files#:~:text=def%20is_ajax(request)%3A